doc/vim-chatgpt.txt

```diff
@@ -43,10 +43,19 @@ Settings                                           *vim-chatgpt-settings*
 Commands:                                           *vim-chatgpt-commands*
 ------------------------------------------------------------------------------
 
-*CodeReviewPlease*  ---  Launch a code review of current buffer.
-			 There are size limitations, see |vim-chatgpt-cli|
-*ChatGPT*           ---  Ask a question to ChatGPT. Ranges are not allowed.
+						*ChatGPT*
+:ChatGPT           ---  Ask a question to ChatGPT. Ranges are not allowed.
 			 For example you could write
 			 :ChatGPT How do I trim a string in golang?
 
+						*:CodeReviewPlease*
+:CodeReviewPlease  ---  Launch a code review of current buffer.
+			 There are size limitations, see |vim-chatgpt-cli|
+
+
+						*:PleaseGenerateTest*
+{visual}PleaseGenerateTest  --- Sends selected block of text to ChatGPT
```

Comment:
perhaps better be all commands had prefix 'chatgpt' for be easy to reference.

Reply:
Yes I do agree, same prefix is more easy to remember than suffix. Maybe ChatGPT...  Or Please... But please gives the impression that we are asking to a human which is weird.